src/main/resources/com/google/api/codegen/java/initcode.snip

```diff
@@ -55,10 +55,15 @@
 @end
 
 @private initLineReadFile(line)
-  String fileName = {@renderInitValue(line.fileName)};
-  Path path = Paths.get(fileName);
-  byte[] data = Files.readAllBytes(path);
-  ByteString {@line.identifier} = ByteString.copyFrom(data);
+  @if line.isFirstReadFileView
```

Comment:
I think it might be worth writing `byteStringFromFile` function and use it. @beccasaurus what do you think?

Reply:
Ping @beccasaurus PTAL :)